teachers_digital_platform/jinja2/teachers_digital_platform/prototypes/crt-survey.html

```diff
@@ -1,148 +1,339 @@
-                    <label class="a-label" for="test_8">Yes</label>
+    <div class="block block__flush-top">
+        <h3 class="h2">Criterion 2: In progress criterion</h3>
+        <div class="lead-paragraph
+                    u-mb45">
+            <p>Does the curriculum address grade-level appropriate topics for earning, income, and careers?</p>
+        </div>
+        <div class="o-survey">
+            <div class="o-survey_number">
+                <div class="h3">2.1</div>
+            </div>
+            <div class="o-survey_indicator">
+                <div class="h3">Indicator</div>
+                <p>Sources of income include wages, salaries, and benefits, as well as interest, rent, and profits.</p>
+            </div>
+            <div class="o-survey_components">
+                <div class="h3">Component</div>
+                <div class="o-survey_component">
+                    <div class="o-survey_question">
+                        <p>Income is money earned from employment and investments.</p>
+                    </div>
+                    <div class="o-survey_answer">
+                        <div class="m-form-field
+                                    m-form-field__radio
+                                    m-form-field__lg-target">
+                            <input class="a-radio" id="crt-question_5a" type="radio" name="crt-question-5" value="1">
+                            <label class="a-label" for="crt-question_5a">
+                                Yes
+                            </label>
+                        </div>
+                        <div class="m-form-field
+                                    m-form-field__radio
+                                    m-form-field__lg-target">
+                            <input class="a-radio" id="crt-question_5b" type="radio" name="crt-question-5" value="0">
+                            <label class="a-label" for="crt-question_5b">
+                                No
+                            </label>
+                        </div>
+                        <div class="m-form-field
+                                    m-form-field__radio
+                                    m-form-field__lg-target">
+                            <input class="a-radio" id="crt-question_5c" type="radio" name="crt-question-5" value="0">
+                            <label class="a-label" for="crt-question_5c">
+                                N/A
+                            </label>
+                        </div>
+                    </div>
                 </div>
-                <div class="m-form-field m-form-field__radio">
-                    <input class="a-radio" type="radio" name="radio_4" id="test_9">
-                    <label class="a-label" for="test_9">No</label>
-                </div>
-            </li>
-            <li>
-                <h4>Indicator</h4>
+            </div>
+        </div>
+        <div class="o-survey">
```

Comment:
I think all `.o-survey` elements should be `li`s of an `<ol class="m-list__unstyled">` for better semantics.

Reply:
Fixed

Reply:
Whoops... this suggestion had an unintended consequence. The `o-survey`s are now narrower than they should be, because we have a standard `max-width` on `.content_main li`. I have to take off a little early today, so I'll leave it up to you what the best way to fix it is.

Reply:
Good catch! I went with `li.o-survey` to make the selector more specific rather than slapping an !important there.